portfolio/src/main/webapp/src/components/BookingForm.js

```diff
@@ -1,75 +1,100 @@
-import React, { Component } from 'react';
-import Button from '@material-ui/core/Button';
+import React, { Component } from "react";
+import Button from "@material-ui/core/Button";
 
-export default class BookingForm extends React.Component{
-    state = {
-        bookingName: '',
-        bookingDate: '',
-        bookingDuration: '',
-        bookingDescription: '',
-        bookingPrice: '',
-    }
-
-    change = (e) => {
-        this.setState({
-            [e.target.name]: e.target.value
-        })    
+export default class BookingForm extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      bookingName: "",
+      bookingDate: "",
+      bookingDuration: "",
+      bookingDescription: "",
+      bookingPrice: "",
     };
+    this.handleChange = this.handleChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
 
-    onSubmit = () => {
-        console.log(this.state);
-    };
+  handleChange(event) {
+    const { name, value, type, checked } = event.target;
```

Comment:
```suggestion
    const {name, value, type, checked} = event.target;
```

Reply:
I think VSCode considers const { name, value, type, checked } the correct formatting since every time I save it, it automatically formats with the spacing.

Reply:
that's fine as well, just be consistent.